minecraft_servers/Lmcraft/manifest.json

```
@@ -0,0 +1,21 @@
+{
+  "server_name": "Lmcraft",
```

`server_name` needs to be lowercased (you need also to rename your created directory)
```suggestion
  "server_name": "lmcraft",
```